data/cookbook/compression/gzip-decompress/00-camlzip.ml

```diff
@@ -0,0 +1,30 @@
+---
+packages:
+- name: "camlzip"
+  version: "1.11"
+libraries:
+- camlzip
+discussion: |
+  - **About `camlzip`:** `camlzip` is a package that brings 3 modules: `Zip`, `Zlib`, and `Gzip`, which permit compression and decompression. The `Gzip` module has two decompression functions, depending on the origin of the compressed stream (file or output channel).
+  - **Reference:** The [`Gzip` `.mli` file](https://github.com/xavierleroy/camlzip/blob/master/gzip.mli) is simple and well documented.
+---
+
+(* The gunzip compression function opens both files (`source` and `dest`) and transfers bytes through a bytes buffer with the appropriate libraries *)
```

Comment:
```suggestion
(* The `Gunzip` compression function opens both files (`source` and `dest`) and transfers bytes through a bytes buffer with the appropriate libraries *)
```
I suspect this needs to be in monspace, but should it also be capitalised like `Gzip` is?